tests/functional/__init__.py

```diff
@@ -3,3 +3,31 @@
 #   This file is distributed under the University of Illinois Open Source
 #   License. See LICENSE.TXT for details.
 # -----------------------------------------------------------------------------
+import os
+import sys
+import json
+
+# setup the required environment variables for the tests
+
+__REPO_ROOT = os.path.abspath(os.environ['REPO_ROOT'])

```

Comment:
I tried to clean up the environment variables, but we need one or two for setting up the PYTHONPATH to find the modules.